content/docs/user-guide/editor/editor-automation.md

```diff
@@ -1,7 +1,8 @@
 ---
-description: ' Use the features of the Python Editor Bindings gem to automate actions
-  and tasks inside of the O3DE Editor. '
-title: Automating the O3DE Editor with the Python Editor Bindings gem
+linkTitle: Automating Editor
+title: Automating the O3DE Editor with the Python Editor Bindings Gem
```

Comment:
question: Related to my earlier 'the' question - here we say 'Automating _the_ O3DE Editor'... (which sounds better to me personally) but elsewhere we have something equivalent to 'Automating O3DE Editor' (no 'the') - should we prefer one over the other or is it okay to mix them interchangeably?